src/pydash/strings.py

```diff
@@ -2052,14 +2126,37 @@ def words(text, pattern=None):
     .. versionchanged:: 3.2.0
         Improved matching for one character words.
     """
-    return reg_exp_js_match(text, pattern or RE_WORDS)
+    if pattern is None:
+        if has_unicode_word(text):
+            return reg_exp_js_match(text, RE_UNICODE_WORDS)
+        else:
+            return reg_exp_js_match(text, RE_WORDS)
+
+    return reg_exp_js_match(text, pattern)
 
 
 #
 # Utility functions not a part of main API
 #
 
 
+def compounder(text):
+    """
+    Remove single quote before passing into words(), required by certain functions such as
+    kebab_case, camel_case, start_case etc.
+
+    to match Lodash-style outputs
+    """
+    return words(deburr(text).replace("'", ""))
```

Comment:
I think we need to remove more than just single-quote.

E.g. https://github.com/lodash/lodash/blob/2f79053d7bc7c9c9561a30dda202b3dcd2b72b90/camelCase.js#L25 removes `/['\u2019]/g` (single-quote and right-single-quote).

But I don't see `deburr` being used in the Lodash implementation. Is there reason to use it here?

Reply:
Good catches. I have included the same regex string used by LodashJs and remove the usage of deburr in commit [cdc8366](https://github.com/dgilland/pydash/pull/159/commits/cdc83665417b0b16105ff544c5cabdeeabc54953)